specifyweb/businessrules/attachment_rules.py

```diff
@@ -25,6 +27,12 @@ def attachment_jointable_save(sender, obj):
     obj.attachment.scopetype, obj.attachment.scopeid = Scoping(attachee)()
     obj.attachment.save()
 
+    if obj.id is None:
```

Comment:
combine these two nested `if`s into a single one?

Reply:
I was primarily following how the other business rules were structured, to stay consistent. 
We should run the backend files through PEP 8 soon...